src/dmd/dtemplate.d

```diff
@@ -6360,8 +6360,7 @@ extern (C++) class TemplateInstance : ScopeDsymbol
          * is behind a conditional debug declaration.
          */
         // workaround for https://issues.dlang.org/show_bug.cgi?id=11239
-        if (global.params.useUnitTests ||
-            global.params.debuglevel)
+        if (global.params.debuglevel)
```

Comment:
Did you test the linked bug after making this change? Doesn't look like the CI gets so far along to run it itself.

Reply:
This is being tested and is passing on the autotester.  It's not until the "test phobos" stage that we get linker errors.

Reply:
@ibuclaw the original bug only had to do with `-debug` switch. As far as I can tell, we have never tried (before this PR) compiling code that doesn't move templates into root modules when `-unittest` is specified.